src/cli/console/input.py

```diff
@@ -3,23 +3,50 @@
 import src.cli.console as console
 
 
+def resolve_duplicates(choices: list, identifiers: list):
+    # detect duplicates
+    duplicates_mask = [True if choices.count(choice) > 1 else False for choice in choices]
+
+    # add identifiers to duplicates
+    choices_resolved = []
+    for choice, identifier, duplicate in zip(choices, identifiers, duplicates_mask):
+        if duplicate:
+            choices_resolved.append(f"{choice} ({identifier})")
+        else:
+            choices_resolved.append(choice)
+
+    return choices_resolved
+
+
 # input
 def list(
-    message,
-    choices,
-    message_no_choices="No choices available!",
+    message: str,
+    choices: list,
+    identifiers: list = [],
```

Comment:
`= None`
e.g.: https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists